src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -368,18 +368,38 @@ def responsive_namespace
 
   def sign_up_link(css_class: nil)
     return unless can_sign_up?
+
     if proxy_mode?
-      link_to('Sign Up', sign_up_params[:url], class: css_class)
+      link_to(sign_up_params[:url], class: css_class) do
+        link_content('Sign Up', css_class, 'fa-user-plus')
+      end
     else
-      link_to('Sign Up', '#', class: css_class, data: { toggle: 'modal', target: '#sign-up-modal' })
+      link_to('#', class: css_class, data: { toggle: 'modal', target: '#sign-up-modal' }) do
```

Comment:
I think we use `'javascript:void(0)'` instead of `#`

Reply:
Well, you use it [here](https://github.com/openSUSE/open-build-service/pull/9192/files#diff-10c13e183f1904c9cf862bd0a498e5fbR6) so you already know it :+1:

Reply:
I just moved code. However, AFAIK, the `'javascript:void(0)'` was added to avoid an unwanted jump/scrolling when opening an offcanvas. In case of _Log In_ and _Sign up_ we open a modal so it is not needed.

Reply:
Ok